### ISSUE TITLE
Blank page on Safari 13 when using `useTheme`

### ISSUE THREAD

**Issue by heiskr** [primer/react#1366]
**Describe the bug**

On Safari version 13 and below, when using `useTheme`, the page appears blank.

I was able to replicate this on docs.github.com as well as https://primer.style/react/Avatar.

<img width="1184" alt="Screen Shot 2021-08-02 at 1 52 29 PM" src="https://user-images.githubusercontent.com/1221423/127922739-14dcfeae-9a47-4a95-a3ce-e882b040e489.png">

**To Reproduce**
1. Find an instance of Safari 13 (I used BrowserStack)
2. Go to https://primer.style/react/Avatar
3. Content will appear temporarily, but the page will empty

**Expected behavior**

The page is visible.

**Screenshots**

<img width="1184" alt="Screen Shot 2021-08-02 at 1 52 29 PM" src="https://user-images.githubusercontent.com/1221423/127922883-b9bc9e18-df1e-40c9-a50f-3aa069331122.png">

**Desktop (please complete the following information):**
 - OS: Mac
 - Browser: Safari
 - Version: 13

**Additional context**

https://github.com/github/docs/issues/8564

**Comment by colebemis** [comment]
Thanks for flagging, @heiskr. I've added this to our tracking board: https://github.com/github/primer/projects/4

**Comment by MykhailoMe** [comment]
@heiskr Yes! The same on Primer. Even with Safari Version 13.1.3 (15609.4.1). MacOS 10.15.7
In Firefox (68.9.0) - Ok.

**Comment by gaknoll** [comment]
For now we are moving this from our inbox to the backlog, but because we haven't heard community feedback on this potential bug, we are going to hold off on investigating further to see if any other customers report it.

**Comment by rachmari** [comment]
@gaknoll @colebemis we've received two more reports of this issue this week. One is here: https://github.com/github/docs-engineering/issues/836 and the other here from a customer using Actions documentation: https://github.slack.com/archives/C0128FB8J3H/p1629163198294500.

Is there a way to prioritize this fix?
